src/components/ArticleCard.jsx

```diff
@@ -1,24 +1,61 @@
-import { Card, Col } from "react-bootstrap";
+import { Card, Col, Button } from "react-bootstrap";
+import { patchArticleVotes } from "../utils/utils";
+import { useState } from "react";
 
 export function ArticleCard({ article }) {
+  const [votes, setVotes] = useState(article.votes);
+  const [error, setError] = useState(false);
   const formattedDate = new Date(article.created_at).toLocaleString();
 
+  const changedVotes = { inc_votes: 0 };
+
+  function handleArticleVote(vote) {
+    setError(false);
+    if (vote === "upvote") {
+      changedVotes.inc_votes = 1;
+    } else {
+      changedVotes.inc_votes = -1;
+    }
+
+    patchArticleVotes(article.article_id, changedVotes)
+      .then((res) => {
+        const { data } = res;
+        setVotes(data.article.votes);
+      })
+      .catch((err) => {
+        setError(true);
+      });
```

Comment:
Currently, users can spam click Up/Downvotes, which isn't great - implement some functionality to stop this!